sqlx-sqlite/src/types/chrono.rs

```diff
@@ -7,7 +9,6 @@ use crate::{
     types::Type,
     Sqlite, SqliteArgumentValue, SqliteTypeInfo, SqliteValueRef,
 };
-use bitflags::_core::fmt::Display;
```

Comment:
Nice catch. For a long while, IntelliJ Rust had a bad habit of using this over the `std` path when auto-importing.

Reply:
Yeah something similar to this happened to this in the RLS days